Packs/Akamai_SIEM/Integrations/Akamai_SIEM/Akamai_SIEM.py

```diff
@@ -169,7 +171,7 @@ def decode_message(msg: str) -> Sequence[str | None]:
     readable_msg = []
     translated_msg = urllib.parse.unquote(msg).split(';')
     for word in translated_msg:
-        word = b64decode(word.encode('utf8')).decode('utf8')
+        word = b64decode(word).decode('utf-8', errors='replace')
```

Comment:
whats this do? we arent expecting it in as bytes anymore?

Reply:
Fixing decoding issues we had, me & @dantavori looked at it together yesterday.